lib/model_renamer/rename.rb

```diff
@@ -52,7 +69,7 @@ def acceptable_filetype? path
   end
 
   def ignore_file? path
-    Array(@opts[:ignore_paths]).any? do |ignore_path|
+    @ignore_paths.any? do |ignore_path|
       path.include? ignore_path
```

Comment:
Flagging this as a potential bug if someone includes 'db' as there ignore path. 

Another option is to go with ignore pattern and use regular expression matching here instead.